apps/ledger-live-desktop/src/renderer/modals/Send/steps/Confirmation/NodeError/index.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from "react";
+import { Flex, Icons, Text } from "@ledgerhq/react-ui";
+import styled, { useTheme } from "styled-components";
+import { CircleWrapper } from "~/renderer/components/CryptoCurrencyIcon";
+import { useExportLogs } from "LLD/hooks/useExportLogs";
+import { createTransactionBroadcastError } from "@ledgerhq/live-common/errors/transactionBroadcastErrors";
+import { urls } from "~/config/urls";
+import { openURL } from "~/renderer/linking";
+import { useLocalizedUrl } from "~/renderer/hooks/useLocalizedUrls";
+import { useCopyToClipboard } from "LLD/hooks/useCopyToClipboard";
+import { useTranslation } from "react-i18next";
+import TechnicalErrorSection from "./TechnicalErrorSection";
+import HelpSection from "./HelpSection";
+import TranslatedError from "~/renderer/components/TranslatedError";
+import { ErrorBody } from "~/renderer/components/ErrorBody";
+
+type Props = {
+  error: Error;
+  currencyName: string;
+  networkName: string;
+};
+
+const InteractIcon = styled(Flex)`
+  &:hover {
+    cursor: pointer;
+  }
+  column-gap: 8px;
+`;
+
+const NodeError = ({ currencyName, networkName, error }: Props) => {
+  const theme = useTheme();
+  const { t } = useTranslation();
+  const { handleExportLogs } = useExportLogs();
+  const { copyToClipboard } = useCopyToClipboard();
+
+  const transactionBroadcastError = createTransactionBroadcastError(error);
+  const url = transactionBroadcastError.url ?? urls.contactSupport;
+  const localizedUrl = useLocalizedUrl(url);
+
+  const [isShowMore, setIsShowMore] = useState(true);
+
+  const color = theme.colors.palette.opacityDefault.c05;
+
+  const onCopyError = () => copyToClipboard(transactionBroadcastError.message);
+
+  const onSaveLogs = () => handleExportLogs();
+
+  const onGetHelp = () => openURL(localizedUrl);
+
+  const onShowMore = () => setIsShowMore(!isShowMore);
+
+  return (
+    <Flex
+      justifyContent="center"
+      alignItems="center"
+      width={"90%"}
+      rowGap={32}
+      flexDirection="column"
+    >
+      <Flex justifyContent="center" flexDirection="column">
+        <ErrorBody
+          title={<TranslatedError error={transactionBroadcastError} />}
+          description={
+            <Text variant="bodyLineHeight" fontSize={14} textAlign="center" color="neutral.c70">
+              {t("errors.TransactionBroadcastError.description", {
+                networkName,
+                currencyName,
+              })}
+            </Text>
+          }
+          top={
+            <CircleWrapper size={72} color={color}>
+              <Icons.DeleteCircleFill size="L" color="error.c50" />
+            </CircleWrapper>
+          }
+        />
+      </Flex>
+      <Flex flexDirection="column" rowGap={16} width="100%" alignItems="flex-start">
+        <Flex alignItems="flex-start">
+          <Text> {t("errors.TransactionBroadcastError.needHelp")}</Text>
+          <InteractIcon onClick={onShowMore}>
```

Comment:
Having to click on the chevron icon specifically is not ideal IMO. Maybe could the click event be on the whole `Flex alignItems="flex-start"` element ?

Reply:
Done ✅